net/https-dns-proxy/files/etc/init.d/https-dns-proxy

```diff
@@ -300,7 +300,9 @@ service_triggers() {
 	local wan wan6 i
 	local procd_trigger_wan6
 	if [ "$on_boot_trigger" = '1' ]; then
+		output "Setting $serviceName raw_trigger for 'interface.*.up'"
 		procd_add_raw_trigger "interface.*.up" 3000 "/etc/init.d/${packageName}" restart 'on_interface_up'
+		output_okn
```

Comment:
Not relevant anyhow to APK migration. If it is not going to be addressed and you will still merge it, I will revert it.